app/test_main.py

```diff
@@ -1,3 +1,58 @@
+import pytest
 from app.main import get_human_age
 
-# write your code here
+
+@pytest.mark.parametrize(
+    "cat_age, dog_age, expected",
+    [
+        (0, 0, [0, 0]),
+        (15, 15, [1, 1]),
+        (14, 14, [0, 0]),
+        (24, 24, [2, 2]),
+        (28, 28, [3, 2]),
+        (100, 100, [21, 17]),
+        (18, 0, [2, 0]),
+        (0, 18, [0, 2]),
+        (15, 24, [1, 2]),
+        (24, 15, [2, 1])
+    ],
+)
+def test_get_human_age(
+    cat_age: int,
+    dog_age: int,
+    expected: list[int]
+) -> None:
+    assert get_human_age(cat_age, dog_age) == expected
+
+
+def test_zero_ages() -> None:
+    assert get_human_age(0, 0) == [0, 0]
+
+
+def test_minimum_cat_and_dog_age() -> None:
+    assert get_human_age(15, 15) == [1, 1]
```

Comment:
The `test_zero_ages` and `test_minimum_cat_and_dog_age` functions are redundant because these cases are already covered by the parameterized test `test_get_human_age`. Consider removing these functions to avoid duplication.